idaes/generic_models/properties/core/coolprop/coolprop_forms.py

```diff
@@ -0,0 +1,226 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+This module contains methods for constructing CoolProp expressions.
+"""
+from pyomo.environ import exp, units as pyunits, Var
+from pyomo.core.expr.calculus.derivatives import Modes, differentiate
+
+from idaes.core.util.exceptions import ConfigurationError
+
+
+# TODO : Only have temperature derivative expression for exponential_tau form
+# TODO : Add other derivative forms as/if required
+
+def parameters_nt_sum(cobj, prop, nlist, tlist):
+    """
+    Method for creating parameters for expression forms using n-t parameters
+
+    Args:
+        cobj - component object that will contain the parameters
```

Comment:
The Google convention is to format arguments like:
```
Args:
   cobj: component object that will contain the parameters
```
The difference being the dash vs. colon. For people, it doesn't matter but I think for Sphinx it will.
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/
Also note that you can add types either as Python 3 type annotations or in parentheses between the argument name and the colonl.